common/src/test/java/com/sequenceiq/cloudbreak/event/ResourceEventTest.java

```diff
@@ -0,0 +1,16 @@
+package com.sequenceiq.cloudbreak.event;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.EnumSource;
+
+class ResourceEventTest {
+
+    @ParameterizedTest(name = "underTest={0}")
+    @EnumSource(ResourceEvent.class)
+    void getMessageTest(ResourceEvent underTest) {
+        assertThat(underTest.getMessage()).isNotNull();
```

Comment:
I think it cannot be null. The test will be much efficient if it loads the related message from the MessageService.

Reply:
As discussed offline, I will have a look (timeboxed) how complicated it would be to cover this with a proper Spring UT.

Reply:
Spring UT has been added; see changes summary above and in the commit message.